packages/use-pixelated-css/src/util/cssUnit.ts

```
@@ -13,7 +13,9 @@ export function convertCSSUnitToPx(value: string, element: HTMLElement): number
     case 'rem':
       return num * parseFloat(getComputedStyle(document.documentElement).fontSize);
     case 'em':
-      return num * parseFloat(getComputedStyle(element).fontSize);
+      const parentElement = element.parentElement;  
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)